R/data.R

```diff
@@ -11,3 +11,13 @@
 #'
 #' @source Constructed using `vs` from the `{pharmaversesdtm}` package and `dm_metabolic` from `{admiralmetabolic}` package
 "vs_metabolic"
+
+#' Example questionnaire dataset
+#'
+#' An example questionnaire SDTM dataset (containing Control of Eating Questionnaire, COEQ, test data) for metabolic studies. Note that University
+#' of Leeds are the copyright holders of the CoEQ and the test data included within `{admiralmetabolic}`is for not-for-profit use only. Any persons
```

Comment:
Does the following imply that people can use the data we provide for non-profit activities? 
"... data included within `{admiralmetabolic}` is for not-for-profit use only."

And if that is the case, do we have confirmation from the copyright holder that this is OK?

Reply:
hmm good point - no they would still need to request a not-for-profit use. What about:

Note that University of Leeds are the copyright holders of the CoEQ and the test data included within `{admiralmetabolic}` is for not-for-profit use only within `{admiralmetabolic}` and pharmaverse-related examples and documentation. Any persons or companies wanting to use the CoEQ should request a license to do so from the following [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).